### PR TITLE
Feat/UI translations

### PR DESCRIPTION
**Pull request by SirZemar**
As we spoke before, events component was handling the logic of the translation. Changed that so now only app component does it. 

However, each component has to import the TranslateService from from ngx-translate/core. Is that ok?

### CODE REVIEW

**Comment by SirZemar** [comment]
To avoid missing tokens, I believe all the languages folders should follow the same model

**Comment by vdias38** [comment]
- Remove lang selector, we'll use browser en as default and set browser lang (`const lang = this.translateService.getBrowserLang();`).
- Remove default angular tokens

File: package.json [hunk 1]
```diff
@@ -68,6 +70,6 @@
   "description": "An Ionic project",
   "lint-staged": {
     "*.ts": "eslint --cache --fix",
-    "*.{ts,scss,html,md}": "prettier --write"
+    "*.{ts,scss,css,md}": "prettier --write"
```
Comment (vdias38) [inline package.json:73]: why remove html file type on prettier processing ?

File: src/app/app.component.ts [hunk 2]
```diff
@@ -1,10 +1,16 @@
...
   templateUrl: 'app.component.html',
   styleUrls: ['app.component.scss'],
 })
 export class AppComponent {
-  constructor() {}
+  constructor(public translate: TranslateService) {
+    translate.addLangs(['en', 'fr', 'pt']);
+    // translate.setDefaultLang('en');
+    const browserLang = translate.getBrowserLang();
+    translate.use(browserLang.match(/en|pt|fr/) ? browserLang : 'en');
```
Comment (vdias38) [inline src/app/app.component.ts:14]: how does it work if lang is 'pt-br' ?
pt-br (langCode-region)
should get only langCode, w/o region

Comment (vdias38) [reply src/app/app.component.ts:14]: use setDefaultLang instead of ternary check

**Comment by SirZemar** [comment]
<h4>Started by rebasing main into the branch.</h4>
```
       $git rebase main
```
<h4>On this rebase process it was necessary to solve conflicts since the first commit of the branch (feat: language select #7691f67).
After solving each commit conflict I add all changes and kept going with rebase process.</h4>
```
	$git add .
	$git rebase —continue
```

Screenshot 2022-06-29 at 16 19 15

Screenshot 2022-06-29 at 16 20 49

<h4>When rebase was complete a sync between local and remote repository was made. </h4>

Joses-MacBook-Pro Eventure josemarinho$ git add

<h4>When rebase was finished, the branch wasn’t in sync with the last master commit, for reason I didn’t understand. It is sync with the commit #55cab36 and it should be with #533e752. </h4>

Screenshot 2022-06-29 at 16 33 56

<h4>After rebase it is asked to merge local branch with remote repository, which feels an unnecessary commit.</h4>

Screenshot 2022-06-29 at 17 05 30

**Comment by SirZemar** [comment]
Do I proceed by pushing these changes to remote repository?

**Comment by vdias38** [comment]
I didn't understand what happens, I suggest to do on a pair session a full PR submit workflow to check any issue.
I've redo a PR from current main https://github.com/devizem/Eventure/pull/60.
We can cancel this one.

**Pull request closed without merging**